.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+  
+jobs:
+  Release:
+    name: Release
+    runs-on: [ubuntu-latest]
+    container:
+      image: iotmod/goreleaser-cgo-cross-compiler:1.13.5
```

Comment:
According to IotMod required to be able to compile the different C impl. of PoW in iota.go for the different operating systems.

Reply:
OK, is this causing any issues with the build system especially since goreleaser is not officially supported with cog? Otherwise we could just use the default go implementation and git rid of all those C dependencies.

Reply:
But definitely in the README we don't need to describe the C dependencies, here people can just to go build and live with the slower performance.

Reply:
I personally would just for consistency's sake with Hornet keep it as it is now. I agree, I wouldn't mention it in the readme. Maybe for whatever unknown reason we will need cgo later on again for other things.

Or do you object and want me to remove it? I don't personally mind that much.

Reply:
My Docker image basically wraps all tools needed to cross compile c (and c++) code for different archs and OS. 
This is needed as the GitHub Action is running on Linux AMD64

Reply:
ok, perfect then!
Let's leave it in then but don't mention cgo in the readme.